File: src/main/java/kitchenpos/apply/order/eatinorders/tobe/infra/JpaEatInOrderRepository.java
package kitchenpos.apply.order.eatinorders.tobe.infra;


import kitchenpos.apply.order.eatinorders.tobe.domain.EatInOrder;
import kitchenpos.apply.order.eatinorders.tobe.domain.EatInOrderRepository;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface JpaEatInOrderRepository extends EatInOrderRepository, JpaRepository<EatInOrder, UUID> {
}

File: src/main/kotlin/kitchenpos/tobe/eatinorder/infra/JpaEatInOrderRepository.kt
package kitchenpos.tobe.eatinorder.infra

import kitchenpos.tobe.eatinorder.domain.entity.EatInOrder
import kitchenpos.tobe.eatinorder.domain.repository.EatInOrderRepository
import org.springframework.data.jpa.repository.JpaRepository

interface JpaEatInOrderRepository : JpaRepository<EatInOrder, Long>, EatInOrderRepository
